zephyr/CMakeLists.txt

```diff
@@ -401,6 +401,14 @@ zephyr_library_sources_ifdef(CONFIG_ZEPHYR_POSIX
 	${SOF_PLATFORM_PATH}/posix/fuzz.c
 )
 
+# When building the fuzzer, the LLVMFuzzerTestOneInput() entry point
+# (which is defined here in the app) must be exposed to the secondary
+# native_sim "runner" linkage step, which normally hides all
+# Zephyr-side symbols to protect against collisions.  Somewhat opaque
+# cmake interface to this, alas.
+set_property(TARGET native_simulator APPEND PROPERTY
```

Comment:
why dont you annotate in the source this function with `NATIVE_SIMULATOR_IF` ? (it is for this)

Reply:
This was the PR that did the change in the sample : https://github.com/zephyrproject-rtos/zephyr/pull/71378
And this is the current sample: 
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/debug/fuzz/src/main.c#L105

Reply:
Nice, didn't notice you already had an API for this.  Updated.